libs/island-ui/core/src/lib/ActionCard/ActionCard.stories.tsx

```diff
@@ -18,6 +19,13 @@ export const Default = () => (
   />
 )
 
+Default.args = {
+  design: {
+    type: 'figma',
+    url: 'https://www.figma.com/file/LKQ4FJ4bTnCSjedbRpk931/Sample-File',
+  },
+}
+
```

Comment:
Is this used?

Reply:
Yes, this is adding a figma tab like this https://pocka.github.io/storybook-addon-designs/?path=/story/docs-figma-examples--embed-file

Reply:
I mean cause the `url` points to some embed sample file, not island.is stuff.